packages/@react-aria/disclosure/src/useDisclosure.ts

```diff
@@ -40,29 +43,47 @@ export interface DisclosureAria {
  * @param state - State for the disclosure, as returned by `useDisclosureState`.
  * @param ref - A ref for the disclosure content.
  */
-export function useDisclosure(props: AriaDisclosureProps, state: DisclosureState, ref?: RefObject<Element | null>): DisclosureAria {
+export function useDisclosure(
+  props: AriaDisclosureProps,
+  state: DisclosureState,
+  ref: RefObject<Element | null>,
+  groupState?: DisclosureGroupState
```

Comment:
Did we discuss doing it the same way as Checkbox and CheckboxGroup? https://github.com/adobe/react-spectrum/blob/main/packages/react-aria-components/src/Checkbox.tsx#L182 since they can either be standalone or in a group as well